threads/src/main/kotlin/pl/edu/pw/ia/threads/application/BannedUserController.kt

```diff
@@ -69,6 +71,8 @@ class BannedUserControllerImpl(
 		webExchange: ServerWebExchange
 	): Mono<IdResponse> {
 		val command = request.toCommand(webExchange.getAccountId())
+		// keycloakService.assignRoleToUser(command.subjectAccountId.toString(), "THREAD_BAN:${command.threadId}")
```

Comment:
this is an example on how to integrate the kc service into existing services, TODO: is to actually use it here/somwhere